src/main/scala/io/citrine/lolo/bags/Bagger.scala

```diff
@@ -76,6 +76,7 @@ case class Bagger(
     )
 
     /* Compute the number of instances of each training row in each training sample */
+    val randBasis = new RandBasis(new ThreadLocalRandomGenerator(new MersenneTwister(rng.nextLong())))
```

Comment:
Add a helper method in this package for getting a breeze `RandBasis` from a `Random` ?

Reply:
Will do